tensorrt_llm/_torch/pyexecutor/kv_cache_manager_v2.py

```
@@ -1356,6 +1357,13 @@ def get_event_window_size(layer_id: int) -> int:
         }
 
     def _format_kv_cache_pool_lifecycle_entry(self, layer_id: LayerId, role: DataRole) -> str:
+        if _cpp_introspection is not None:
+            lifecycle_id = self.impl.get_layer_group_id(layer_id)
+            layer = self.kv_cache_manager_py_config.layers[int(layer_id)]
+            return (
+                f"role={str(role)}, lifecycle_id={int(lifecycle_id)}, "
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)